modules/ROOT/pages/cloudhub-networking-guide.adoc

```diff
@@ -45,7 +46,7 @@ If other values for `http.port` and `https.port` are specified in the `mule-app.
 
 [NOTE]
 --
-For each request a client makes through CloudHub's load balancer (`myapp.cloudhub.io`), the load balancer maintains two connections. One connection ​is from​ the client and one is to your worker.  For each connection, the load balancer manages an idle timeout of 300 seconds that is triggered when no data is sent over ​either​ connection. If no data is sent or received during this time period, the load balancer closes ​both connections.
+For each request a client makes through CloudHub's load balancer (`myapp.region.cloudhub.io`), the load balancer maintains two connections. One connection ​is from​ the client and one is to your worker.  For each connection, the load balancer manages an idle timeout of 300 seconds that is triggered when no data is sent over ​either​ connection. If no data is sent or received during this time period, the load balancer closes ​both connections.
```

Comment:
About "and one is to your worker": shouldn't "worker" be plural? After all, a load balancer balances requests among more than one worker, right?
About "the load balancer manages an idle timeout of 300 seconds that is triggered...": The timeout is "idle"? And does the load balancer "manage an idle timeout...that is triggered", or does it simply close connections when they have been idle for 300 seconds?